docker/php-dockerfile

```diff
@@ -1,6 +1,12 @@
-FROM php:5.6.14-fpm
+# Targeting 5.6 instead of a specific version is consistent with our current Chef setup
+# which in turn targets "php5.6-*" packages in the ondrej repo.
+FROM php:5.6-fpm
 EXPOSE 9000
 COPY './docker/php-fpm.conf' "/usr/local/etc/php-fpm.conf"
 COPY './docker/php.ini' "/usr/local/etc/php/php.ini"
 COPY './docker/php-mysqli.ini' "/usr/local/etc/php/conf.d/docker-php-ext-mysqli.ini"
-RUN docker-php-ext-install mysqli mbstring pdo pdo_mysql
+RUN docker-php-ext-install mysql mysqli mbstring pdo pdo_mysql
+# configure mysql bindings to use mysqlnd service, which is consistent with our current Chef setup
+RUN docker-php-ext-configure pdo_mysql --with-pdo-mysql=mysqlnd
+RUN docker-php-ext-configure mysql --with-pdo-mysql=mysqlnd
```

Comment:
This is basically trying to mimick what `apt install php5.6-mysqlnd-ms` does, except for the `ms` part. I just reconfigured all adapters because our [current phpinfo](https://worldcubeassociation.org/results/admin/phpinfo.php) suggests that all adapters are running under `mysqlnd` in the current setup.